app/lib/config/config.go

```diff
@@ -237,3 +239,16 @@ func (s *ServerToml) String() string {
 	}
 	return buff.String()
 }
+
+// TildeToHome takes a path and replaces an eventual "~" with the home-directory
+func TildeToHome(path string) string {
+	if strings.HasPrefix(path, "~") {
+		usr, err := user.Current()
+		if err != nil {
+			log.Error("Could't get home-directory")
```

Comment:
Ah you changed it back to `Error` ? Hehe. I was more OK with `Fatal` because if you can't get the home directory, you can't do much, and the rest of the program should not continue working on some invalid data ... What do you think ?